client/python/lib/ovmsclient/tfs_compat/serving_client.py

```diff
@@ -0,0 +1,230 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+class ServingClient:
+    
+    def __init__(self, address, grpc_port, http_port=None):
+        raise NotImplementedError
+
+    def send_predict_request(self, request):
+        '''
+        Send PredictRequest to the server.
+
+        Args:
+            request: PredictRequest object. If the request is of type GrpcPredictRequest
```

Comment:
I'm not sure if this is good idea to make universal predict method because REST and gRPC will use different connection type. Maybe we should have two classes for connections: for REST and for gRPC. It might be useful to control which grpc stub will be used for which predictions while for REST it is not the case.

Reply:
Could you elaborate on "It might be useful to control which grpc stub will be used for which predictions" part?
Connection objects for both HTTP and GRPC will be held inside ServingClient. User will create predict request of certain type and based on that type ServingClient will dispatch it via appropriate interface.